generator/feature_builder.hpp

```diff
@@ -302,4 +372,35 @@ void ForEachParallelFromDatRawFormat(size_t threadsCount, std::string const & fi
   for (size_t i = 0; i < threadsCount; ++i)
     threadPool.Submit(concurrentProcessor);
 }
+template <class SerializePolicy = serialization_policy::MinSize>
+std::vector<FeatureBuilder> ReadAllDatRawFormat(std::string const & fileName)
+{
+  std::vector<FeatureBuilder> fbs;
+  ForEachFromDatRawFormat<SerializePolicy>(fileName, [&](auto && fb, auto const &) {
+    fbs.emplace_back(std::move(fb));
+  });
+  return fbs;
+}
+
+template <class SerializePolicy = serialization_policy::MinSize, class Writer = FileWriter>
+class FeatureBuilderWriter
+{
+public:
+  FeatureBuilderWriter(std::string const & filename, FileWriter::Op op = FileWriter::Op::OP_WRITE_TRUNCATE)
+    : m_writer(filename, op)
+  {
+//    WriteVarUint(m_writer, static_cast<serialization_policy::TypeSerializationVersion>(SerializePolicy::kSerializationVersion));
```

Comment:
add a comment why this is commented out, please

Reply:
++